feasibility-triangle/troubleshooting/cql/report.jq

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This does not work on my version of blaze 0.27.1
as the response from blazectl (OUTPUT in the test-cql.sh) looks as follows:

{"date":"2024-06-25T06:45:12.855811506Z","group":[{"population":[{"count":0,"code":{"coding":[{"system":"http://terminology.hl7.org/CodeSystem/measure-population","code":"initial-population"}]}}]}],"type":"summary","resourceType":"MeasureReport","measure":"urn:uuid:de9b855a-4415-44d4-ab56-f777331f85b4","extension":[{"url":"https://samply.github.io/blaze/fhir/StructureDefinition/eval-duration","valueQuantity":{"value":0.064653792,"unit":"s","system":"http://unitsofmeasure.org","code":"s"}}],"status":"complete","period":{"start":"1900","end":"2200"}}

Reply:
note it works if i only leave the result: .... part of the file

Reply:
Yes I wrote it only for the upcoming Blaze version with CQL cache. Now it works also for older versions.